met-web/src/components/layout/Table/Table.tsx

```diff
@@ -205,7 +204,7 @@ function EnhancedTable() {
         setDense(event.target.checked);
     };
 
-    const isSelected = (id: number) => selected.indexOf(id) !== -1;
+    const isSelected = (id: number) => selected.indexOf(Number(id)) !== -1;
```

Comment:
probably we should look at way of handling Number(id); we are doing it eveyrwhere..
may be make id string itself or somethng like that.. doing Number(id)is no fun